webrtc.html

```diff
@@ -450,6 +451,15 @@
                 <code><a data-link-for=
                 "RTCPeerConnection">localDescription</a></code> attribute.</p>
               </dd>
+              <dt><dfn><code>reofferCodecs</code></dfn> of type <span class=
+              "idlMemberType"><a>boolean</a></span>, defaulting to
+              <code>false</code></dt>
+              <dd>
+                <p>When the value of this dictionary member is true,
+                the generated description will include codecs even if
+                they were excluded by the most recent answer, as
+                defined in [[!JSEP]].</p>
```

Comment:
this should link to a specific section of JSEP with `<span data-jsep=section-id>[[!JSEP]]</span>`.